pages/ksphere/kubeflow/1.0.1-0.4.0/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+navigationTitle: KUDO for Kubeflow Technical Preview
+title: Welcome to KUDO for Kubeflow Technical Preview
+beta: false
+menuWeight: 1
+---
+
+![KUDO for Kubeflow](img/d2kf-overview.png)
```

Comment:
I built the docs site, but I don't see this page anywhere in the resulting site.  Where is it supposed to be?

Reply:
Hm the page is there, as it [is for the previous version](https://docs.d2iq.com/ksphere/kubeflow/1.0.1-0.3.1/). It just does not seem to be linked to from anywhere? Perhaps it only gets linked to if there is more than one version of kubeflow present? @cmastr do you know?
It's a pity that the nobody gets to see the diagram in that case. cc @akirillov

Reply:
Looking at this. Will see if I can add it to the TOC.

Reply:
Any luck with this? If not, then perhaps we can postpone this to later? We had a similar unreachable page in the previous release as well...

Reply:
Yeah I didn't get very far with this so we'll have to hold off for now. There would have to be some restructuring of the content in the Intro page.